index.d.ts

```diff
@@ -54,8 +98,39 @@ console.log(result);
 //=> ['https://sindresorhus.com/', 'https://avajs.dev/', 'https://github.com/']
 ```
 */
-export default function pMap<Element, NewElement>(
-	input: Iterable<Element>,
-	mapper: Mapper<Element, NewElement>,
-	options?: Options
-): Promise<NewElement[]>;
+declare const pMap: {
+	<Element, NewElement>(
+		input: Iterable<Element>,
+		mapper: Mapper<Element, NewElement>,
+		options?: Options
+	): Promise<NewElement[]>;
+
+	/**
+	Creates a special object that indicates to `pMap` that iteration must stop immediately. This object should just be returned from within the mapper (and not used directly for anything).
+
+	@example
+	```
+	import pMap from 'p-map';
+	import got from 'got';
+
+	const numbers = Array.from({ length: 2000 }).map((_, i) => i + 1);
```

Comment:
```suggestion
	const numbers = Array.from({length: 2000}).map((_, index) => index + 1);
```

Same in other places too.

Reply:
Done: e70cd78